### PR TITLE
Upgrade to solidus 3 x

### PR DESCRIPTION
**Pull request by DanielePalombo**

### CODE REVIEW

File: spec/models/solidus_zip_zones/tax_location_decorator_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'spec_helper'
 
 RSpec.describe SolidusZipZones::TaxLocationDecorator do
-  subject { Spree::Tax::TaxLocation.new }
+  subject(new_tax_location) { Spree::Tax::TaxLocation.new }
```
Comment (elia) [inline spec/models/solidus_zip_zones/tax_location_decorator_spec.rb:4]: missing colon?

**Review by elia — CHANGES_REQUESTED** [review]
looks good, just noted a typo that might be causing the CI to fail

**Review by cpfergus1 — APPROVED** [review]
Looks good to me, I added that colon to help push things along!

File: spec/models/solidus_zip_zones/tax_location_decorator_spec.rb (hunk 1 above)
Comment (cpfergus1) [reply spec/models/solidus_zip_zones/tax_location_decorator_spec.rb]: I got this in 3d1c828

**Pull request merged**